cudnn_att.cu

```diff
@@ -30,28 +34,17 @@ static void cudaCheck(cudaError_t error, const char *file, int line) {
 #define cudaCheck(err) (cudaCheck(err, __FILE__, __LINE__))
 
 // Profiler utils
-namespace {
-    class NvtxRange {
-    public:
-        NvtxRange(const char* s) { nvtxRangePush(s); }
-
-        NvtxRange(const std::string& base_str, int number) {
-            std::string range_string = base_str + " " + std::to_string(number);
-            nvtxRangePush(range_string.c_str());
-        }
-
-        ~NvtxRange() { nvtxRangePop(); }
-    };
-}
+class NvtxRange {
```

Comment:
this seems like an  ODR violation waiting  to happen?

Reply:
Oops, fixed - I copy-pasted that as part of a chunk from train_gpt2.cu, didn't notice the namespace you added, thanks!